test/utils/fieldUtils.js

```diff
@@ -93,24 +93,24 @@ class FieldDataTypes {
   }
 
   /**
-   * Select a random option from the dropdown
+   * Select a provided option from the dropdown
```

Comment:
or random

Reply:
perhaps we should explicitly state in the doc here that if the value parameter is left blank then a random option will be chosen